dev/com.ibm.ws.concurrent.persistent_fat_compat/fat/src/com/ibm/ws/concurrent/persistent/fat/compat/PersistentExecutorCompatibilityTest.java

```diff
@@ -0,0 +1,233 @@
+
+/**
+ * Tests for persistent scheduled executor with task execution disabled
+ */
+@MinimumJavaLevel(javaLevel = 7)
+public class PersistentExecutorCompatibilityTest {
+    private static final LibertyServer server = FATSuite.server;
+    
+    public static final String APP_NAME = "persistentcompattest";
+
+    @Rule
+    public TestName testName = new TestName();
+
+    /**
+     * Runs a test in the servlet.
+     * 
+     * @param test Test name to supply as an argument to the servlet
+     * @return output of the servlet
+     * @throws IOException if an error occurs
+     */
+    protected static StringBuilder runInServlet(String test) throws Exception {
+        URL url = new URL("http://" + server.getHostname() + ":" + server.getHttpDefaultPort() + "/persistentcompattest?test=" + test);
+        Log.info(PersistentExecutorCompatibilityTest.class, "runInServlet", "URL is " + url);
+        HttpURLConnection con = (HttpURLConnection) url.openConnection();
+        try {
+            con.setDoInput(true);
+            con.setDoOutput(true);
+            con.setUseCaches(false);
+            con.setRequestMethod("GET");
+            InputStream is = con.getInputStream();
+            InputStreamReader isr = new InputStreamReader(is);
+            BufferedReader br = new BufferedReader(isr);
+
+            String sep = System.getProperty("line.separator");
+            StringBuilder lines = new StringBuilder();
+
+            // Send output from servlet to console output
+            for (String line = br.readLine(); line != null; line = br.readLine()) {
+                lines.append(line).append(sep);
+                Log.info(PersistentExecutorCompatibilityTest.class, "runInServlet", line);
+            }
+
+            // Look for success message, otherwise fail test
+            if (lines.indexOf("COMPLETED SUCCESSFULLY") < 0) {
+                Log.info(PersistentExecutorCompatibilityTest.class, "runInServlet", "failed to find completed successfully message");
+                fail("Missing success message in output. " + lines);
+            }
+
+            return lines;
+        } finally {
+            con.disconnect();
+            Log.info(PersistentExecutorCompatibilityTest.class, "runInServlet", "disconnected from servlet");
+        }
+    }
+
+    /**
+     * Before running any tests, start the server
+     * 
+     * @throws Exception
+     */
+    @BeforeClass
+    public static void setUp() throws Exception {
+    	//Create app to be tested against.
+    	WebArchive app = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+                .addPackage("web")
+                .addPackage("ejb")
+                .addAsWebInfResource(new File("test-applications/" + APP_NAME + "/resources/WEB-INF/web.xml"));
+    	
+    	//Copy contents of serialized directory to WEB-INF directory
+    	String serDir = "test-applications/" + APP_NAME + "/resources/WEB-INF/serialized";
+    	app.merge(
```

Comment:
This app.merge method was used to get the WEB-INF/serialized/ directory and files into the war.  Is there a better way?